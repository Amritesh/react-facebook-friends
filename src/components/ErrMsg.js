import React, { Component } from 'react';

class ErrMsg extends Component {

  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>ERR</div>
    );
  }
}

export default ErrMsg;
