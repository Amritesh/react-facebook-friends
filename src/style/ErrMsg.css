.wrapper {
  min-height: 50vh;
  display: flex;
  justify-content: center;
  align-items: center;
}

.box {
  border-radius: 3px;
  background: #FFF6F6;
  padding: 15px;
}

.title {
  font-size: 20px;
  text-align: center;
  color: #912D2B;
  font-weight: bold
}

.msg {
  color: rgb(159, 58, 56);
}
