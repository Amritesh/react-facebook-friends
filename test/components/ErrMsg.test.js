import React from 'react';
import renderer from 'react-test-renderer';
import ErrMsg from '../../src/components/ErrMsg';

test('ErrMsg component should renders correctly', () => {
  const component = renderer.create(<ErrMsg />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
