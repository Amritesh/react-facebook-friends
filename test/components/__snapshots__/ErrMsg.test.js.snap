// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrMsg component should renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="box"
  >
    <div
      className="title"
    >
      Error
    </div>
    <p
      className="msg"
    >
      We are sorry but something went wrong, please try again later.
    </p>
  </div>
</div>
`;
